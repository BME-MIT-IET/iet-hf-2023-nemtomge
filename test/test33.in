loadmap map.json
start V1 V2  
set Scientist V1 PL1 
set Scientist V2 PL2 
give Active V1 Bear 1 
move V2 PL1 # V2 to moves to the same field as V1  
has Active V2 Bear
exit
