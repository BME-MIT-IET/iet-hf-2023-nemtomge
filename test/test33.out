Map from file map.json loaded
The game has been started. Players are: V1 V2  
Scientist V1 set on field PL1 
Scientist V2 set on field PL2 
Scientist V1 has been given active agent Bear time: 1 
Scientist V2 moved to field PL1
Scientist V2 has active agent Bear time: -1
